community/2024-08-07-community-meeting.mdx

```diff
@@ -0,0 +1,60 @@
+---
+date: '2024-08-07'
+title: 'Meeting Agenda - 2024-08-07'
+---
+
+## Agenda
+
+{/* Update before the start of the meeting with planned agenda items */}
+
+-
+-
+-
+
+{/* truncate */}
+
+## Meeting Notes
+
+{/* Add summary of topics during the meeting. Each Topic should have an H3 heading. A list is usually sufficient for the recap */}
+
+### Topic 1
+
+- Note 1
+- Note 2
+
+### Topic 2
+
+- Note 1
+- Note 2
+
+## Weekly highlights
+
+### Issue of the Week
+
+- [title](github.com/wasmcloud/wasmcloud/issues/)
```

Comment:
```suggestion
- [title](https://github.com/wasmcloud/wasmcloud/issues/)
```

This is causing the build to fail because technically ☝️🤓 this isn't a valid link (my bad)